web_scraper.py

```diff
@@ -1,65 +1,72 @@
 from bs4 import BeautifulSoup
 import requests
 
+
 def parse_pet(href):
     html = requests.get(href)
     html_text = html.text
     soup = BeautifulSoup(html_text, 'lxml')
-    details = soup.find('ul', class_ = 'petdetails')
+    details = soup.find('ul', class_='petdetails')
     indents = details.find_all('li')
-    #W poniższych zmiennych zapisze dane, None oznacza brak danych
-    age = None
-    breed = None
-    gender = None
-    weight = None
-    number = None
-    status = None
-    date_in = None
-    date_out = None
-    box = None
-    address_found = None
-    group_link = None
-    group_name = None
+    # W poniższym słowniku zapisze dane, None oznacza brak danych
+    info = {

```

Comment:
Please rename to animal_info or something similiar so that it is self explanatory

Reply:
> I still think animal info should be a Python Class rather then just a dictionary

Because all animals will be stored in a database with a Django model I do not think that using a class is necessary. The model itself will be a class, which will be very similar to a suggested animal class.